go.mod

```diff
@@ -159,3 +161,5 @@ replace (
 	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.19.3
 	k8s.io/sample-controller => k8s.io/sample-controller v0.19.3
 )
+
+replace github.com/banzaicloud/integrated-service-sdk => ./lib/integrated-service-sdk
```

Comment:
This should be moved to the replace block above.

Reply:
thx, fixed